### PR TITLE
Added the Quick Tutorial for Angular 

### LINKED ISSUE #25: Add Angular Starter Template
**I'm submitting a ...** 

- [ ] bug report
- [x] feature request

**Current behavior:**

There is no starter template for Angular.
**Expected behavior:**

I'll add the starter template for Angular, by which one can easily get a quick tutorial for the same.
**Other information:**

### PR DESCRIPTION
**Pull request by ghost**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for new templates)
- [x] Docs have been reviewed and added / updated if needed (for new templates)

## Fixes issue

 

Fixes: #25 

## Pull request type

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

- There is no quick tutorial for Angular
-

## What is the new behavior?

- Added the quick tutorial for Angular.
-

## Other information

Will add the template soon.

### CODE REVIEW

**Comment by ghost** [comment]
Quick Tutorial of Angular is done! @harsh-2711 
Please review.

**Comment by harsh-2711** [comment]
@waterupto You are only adding very basic steps. It will be good if you can first provide some started template and then write about the tech. Because AFAIK, this will be overwritten by some contributor who will actually write the starter template. So, do you want to create a started template or should I merge it?

**Comment by ghost** [comment]
You may merge it if you feel like, @harsh-2711 sir. I'll definitely make the template and upload it as soon as possible.

File: frontend/angular/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+## Introduction
+Angular is a popular front-end framework made by Google. Like other popular front-end frameworks, it uses a component-based architecture to structure apps.
+
+If you are a beginner for Angular and wonder how to start, This article is for you and we focus only on basic Angular App with one page. Once finished our app looks like below where we can add items, edit, delete items.
+
+## How to get started with Angular
+
+-   Install [Nodejs](https://nodejs.org/en/download/)
+-   Install Angular CLI
+-   Install Typescript globally
```
Comment (harsh-2711) [inline frontend/angular/README.md]: Don't do duplication. You can combine this with the command to install it.

File: frontend/angular/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    npm install -g @angular/cli
+    npm install -g typescript 
+  
+## Goal
+-   Provide quick start template for projects with flavour of Bootstrap.
+-   Provide a range of features that are common to large classes of projects.
+
+## Pre-requisites
+-   Basic understanding of Typescript/Javascript
+-   Basic knowledge about Angular
```
Comment (harsh-2711) [inline frontend/angular/README.md]: Can't add this in pre-requisite. The whole idea of the repo is that we provide them with some basic knowledge of any tech to start with along with a template and some links to advance tutorials.

File: frontend/angular/README.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+-   Install Typescript globally
+
+  Once you install nodejs and we need to install Angular cli and typescript globally with the following commands.
+  
+    npm install -g @angular/cli
+    npm install -g typescript 
+  
+## Goal
+-   Provide quick start template for projects with flavour of Bootstrap.
+-   Provide a range of features that are common to large classes of projects.
```
Comment (harsh-2711) [inline frontend/angular/README.md]: Try to mention explicitly which features.

**Review by harsh-2711 — CHANGES_REQUESTED** [review]

File: frontend/angular/README.md (hunk 1 above)
Comment (ghost) [reply frontend/angular/README.md]: Done sir, please review the same @harsh-2711

File: frontend/angular/README.md (hunk 2 above)
Comment (ghost) [reply frontend/angular/README.md]: I have deleted the pre-requisites taking into what you had said above. Please review @harsh-2711

File: frontend/angular/README.md (hunk 3 above)
Comment (ghost) [reply frontend/angular/README.md]: Tried to explain this as well in detail. Please review @harsh-2711

File: frontend/angular/README.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
+## Introduction
+Angular is a popular front-end framework made by Google. Like other popular front-end frameworks, it uses a component-based architecture to structure apps.
+
+If you are a beginner for Angular and wonder how to start, This article is for you and we focus only on basic Angular App with one page. Once finished our app looks like below where we can add items, edit, delete items.
```
Comment (harsh-2711) [inline frontend/angular/README.md]:
```suggestion
If you are a beginner for Angular and wonder how to start, this article is for you and we focus only on basic Angular App with one page. Once finished our app looks like below where we can add items, edit, delete items.
```

Which finished app are you talking about?

**Review by harsh-2711 — CHANGES_REQUESTED** [review]

File: frontend/angular/README.md (hunk 4 above)
Comment (ghost) [reply frontend/angular/README.md]: I mixed something else here. I have rectified it. Please review, sir.

**Comment by ghost** [comment]
@harsh-2711 There's only this one problem that I am not able to rectify, could you please help me with it?

**Comment by harsh-2711** [comment]
@waterupto These changes are good to go but we can't merge this incomplete thing. I request you to make a branch out of it and work on adding Angular starter template along with a brief description. We can merge it then.

**Pull request closed without merging**